### PR TITLE
68 design proposals low effort

### PR DESCRIPTION
**Pull request by juanjoseortega-aircury**

### CODE REVIEW

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
Just pending to ask the client if they are happy with the font you chose. Maybe even providing them with different proposals would be nice.

The rest looks good.

File: assets/js/img_clicker.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ $(function () {
         document.getElementById("descPopUp").textContent = $description;
         document.getElementById("namePopUp").textContent = $name;
         document.getElementById("jobPopUp").textContent = $job;
+        document.get
```
Comment (cristobaldiaz-aircury) [inline assets/js/img_clicker.js:12]: I'm guessing you left this by accident

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
1. There are many scss variables that after the changes aren't used anymore. As it seems like there are many of them, create a ticket to solve the problem.
2. Look for a way to use the fonts without adding them to the repository. It can be done as a separate ticket.

**Comment by juanjoseortega-aircury** [comment]
> 1. There are many scss variables that after the changes aren't used anymore. As it seems like there are many of them, create a ticket to solve the problem.
> 2. Look for a way to use the fonts without adding them to the repository. It can be done as a separate ticket.

The variables can be deleted at the ticket for cleaning unused code.
About the fonts, only Julius Sans can be used from google fonts but this downgrade the performance

**Review by cristobaldiaz-aircury — APPROVED** [review]

**Pull request merged**